src/constants.rs

```diff
@@ -0,0 +1,13 @@
+pub const RPC: &str = "beta-5.fuel.network";
+
+pub const TOKEN_CONTRACT_ID: &str =
+    "0x3bcfb9d38a65fd39a829e5137ebf4bb2a61429d319a2899e457fce0b436ca1a6";
+pub const USDC_ASSET_ID_STR: &str =
+    "0x0450e4d385cbd2914f74505f18f01587cc4f4ad1fdef4b80cbde2a8155a86d72";
+pub const BTC_ASSET_ID_STR: &str =
+    "0x593b117a05f5ea64b39ba1f9bc3fb7e7a791c9be130e28376ad552eacdb3b746";
+pub const UIN_ASSET_ID_STR: &str =
```

Comment:
Is this `UIN` or `UNI`?

Reply:
@chlenc

Reply:
This is UNI token that was issued used our token contract 
https://github.com/compolabs/src-20